components/bl-rating-component/styles/index.less

```diff
@@ -0,0 +1,54 @@
+// you can use LESS variables from the current Theme and app extensions of the UI-Container
+@bl-customComponent-badge-themeColor: @themePrimary;
```

Comment:
Could we add variables for disabled icons?

Reply:
Sure